src/pages/EditMonitor.vue

```diff
@@ -352,20 +352,6 @@ export default {
             return this.$root.baseURL + "/api/push/" + this.monitor.pushToken + "?msg=OK&ping=";
         },
 
-        bodyPlaceholder() {
-            return `Example:
-{
-    "key": "value"
-}`;
-        },
-
-        headersPlaceholder() {
-            return `Example:
-{
-    "HeaderName": "HeaderValue"
-}`;
-        }
-
```

Comment:
I prefer not to translate the json. Just need to translate "Example:"

Reply:
I reverted these changes, used the existing translation for "Example:" and kept the JSON placeholders as they were in `EditMonitor.vue`.